api/src/utils/media/dwc/dwc-archive-file.ts

```diff
@@ -87,6 +87,16 @@ export class DWCArchive {
     }
   }
 
+  _workbookFromWorksheets() {
```

Comment:
Could do yeah. I can't remember the original thinking here, but it was probably because the DWC files are all separate csv files, so I converted them each into a worksheet, but never worried about combining them into a workbook since we never do anything that the workbook level, before now any ways.